pages/api/upload_album.py

```
@@ -25,13 +25,15 @@
 from lib.database import execute_query
 from pathlib import Path
 import traceback
+from pages.api.shares import *
```

Ruff (0.7.0)

`from pages.api.shares import *` used; unable to detect undefined names

(F403)